index.js

```diff
@@ -573,8 +574,23 @@ class Raspi extends EventEmitter {
     throw new Error('reportDigitalPin is not yet implemented');
   }
 
-  pingRead() {
-    throw new Error('pingRead is not yet implemented');
+  pingRead(config, handler) {
+    const pulseOut = config.pulseOut;
```

Comment:
Look at Particle-IO, it just ignores everything except for `pin` property: https://github.com/rwaldron/particle-io/blob/master/lib/particle.js#L544-L565